hail/python/test/hail/test_ir.py

```diff
@@ -188,6 +188,7 @@ def test_matrix_ir_parses(self):
         table_read = ir.TableRead(resource('backward_compatability/1.0.0/table/0.ht'), False, None)
 
         matrix_irs = [
```

Comment:
You also need to add this to IRSuite and maybe MatrixIRSuite as well to test it's actually working.